harmony-api/src/routes/follow/followCount/followCountRouter.ts

```diff
@@ -5,5 +5,8 @@ import * as followCountController from '../../../controllers/follow/followCount/
 const router = express.Router()
 
 router.get('/', followCountController.getFollowCount)
+router.get('/artist', followCountController.getArtistFollowCount)
```

Comment:
Regarding moving the id’s from headers to params, would probably make sense to make the route something like /artist/:artistId